source/coastal_vulnerability.rst

```diff
@@ -61,7 +61,7 @@ Table 4.1
 """""""""
 
 .. csv-table:: **Example Ranking System**
-      :file: ./coastal_vulnerability/cv_table_41.csv
+      :file: ./coastal_vulnerability/ranking_system_table.csv
```

Comment:
This looks great. I am seeing `Table 8` instead of the before `Table 4`. The `Table 4.1: List ...` then doesn't line up... Not sure what's throwing off table numbers.